app/containers/message/Video.tsx

```diff
@@ -38,10 +38,11 @@ interface IMessageVideo {
 	getCustomEmoji: TGetCustomEmoji;
 	style?: StyleProp<TextStyle>[];
 	isReply?: boolean;
+	msg?: string;
 }
 
 const Video = React.memo(
```

Comment:
Because isn't work properly for Video and Audio

Reply:
Why wouldn't it work? Did you get to test it?
Because from what I understand you're just looking, code, if you had tested it you wouldn't have come to that conclusion.

Reply:
I tested both, also this React.memo is doing the propsAreEqual based only on props.file, and the msg is another prop. The same is for Audio.

Reply:
@reinaldonetof tips for the next reports, make the report clearer, saying that it doesn't work and how to get to the error.
In this case, it works partially, but it doesn't work if you do a specific behavior.
Just saying that it doesn't work doesn't help the person to correct the mistake, it also increases the rework time, since the person will have to find out what doesn't work.